app/javascript/src/components/Dashboard/Notes/NoteItem/NoteItem.jsx

```diff
@@ -44,9 +45,9 @@ const NoteItem = ({
         />
         <div className="flex items-center">
           <Clock color="#1e1e20" size={22} />
-          <Tooltip content="Wednesday, 10:30AM" position="bottom">
+          <Tooltip content={formatDate(createdAt)} position="bottom-end">
             <Typography className="mx-2" style="body3">
-              {updatedAt}
+              {lastUpdated(createdAt)}
```

Comment:
Very confusing util name when we read it along with the argument passed to it. Do you feel a disparity here?